pro/src/components/pages/Home/Offerers/Offerers.jsx

```diff
@@ -58,7 +64,7 @@ const Offerers = () => {
         }
       })
     },
-    [query]
+    [offererId]
```

Comment:
J'ai l'impression qu'on refait le call api, alors que seul le traitement des données change quand `offererId` change, mais les données récupérées sont les mêmes. Quel est le cas d'usage (cela correspond à quoi dans l'interface) ? est-ce que ça arrive souvent ?

Reply:
tu peux mettre un commentaire pour qu'on investigue plus tard si ça a un impact (si c'est effectivement appelé plusieurs fois) ? `listOfferersNames()` renvoie toujours la même chose indépendamment de `offererId`, il n'y a pas de raison de le rappeler
par ex `// TO FIX: avoid calling listOfferersNames() again when offererId changes`